app/services/open_feature/dev_cycle/store.py

```
@@ -0,0 +1,415 @@
+                host=REDIS_HOST,
+                port=int(REDIS_PORT),
+                decode_responses=True,
+                socket_timeout=1.0,  # Quick timeout for sync calls
+                socket_connect_timeout=1.0,
+            )
+            value_str = client.get(f"{FEATURE_FLAGS_PREFIX}{key}")
+            if value_str:
+                import json
+
+                return json.loads(value_str)
+        return None
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)